source/MaterialXView/Material.h

```diff
@@ -164,6 +164,24 @@ class Material
     /// Find a public uniform from its MaterialX path.
     mx::ShaderPort* findUniform(const std::string& path) const;
 
+    /// Change the uniform value inside shader and MaterialX doc.
```

Comment:
Minor grammar: "inside the shader and the associated element in the MaterialX document."